tests/base/src/test/java/org/wildfly/security/auth/server/CustomNameRewriterTest.java

```diff
@@ -0,0 +1,75 @@
+package org.wildfly.security.auth.server;
```

Comment:
@lvydra Just a minor, can you please add a license on top of the file? Thank you!

Reply:
@Skyllarr Added